sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -114,6 +129,24 @@ def test_roc_curve_display_plotting(
     assert isinstance(display.ax_, mpl.axes.Axes)
     assert isinstance(display.figure_, mpl.figure.Figure)
 
+    if plot_chance_level:
+        assert isinstance(display.chance_level_, mpl.lines.Line2D)
+        assert tuple(display.chance_level_.get_xdata()) == (0, 1)
+        assert tuple(display.chance_level_.get_ydata()) == (0, 1)
+
+        if "color" not in chance_level_kwargs:
+            assert display.chance_level_.get_color() == "k"
+        if "linestyle" not in chance_level_kwargs:
+            assert display.chance_level_.get_linestyle() == "--"
+        if "label" not in chance_level_kwargs:
+            assert display.chance_level_.get_label() == "Chance level (AUC = 0.5)"
+
+        for k, v in chance_level_kwargs.items():
+            assert getattr(display.chance_level_, "get_" + k)() == v
+
+    else:
+        assert display.chance_level_ is None
+
```

Comment:
Let's isolate this part in a new test that will be specific to check the behaviour of `plot_chance_line` and `chance_line_kwargs`.